src/Toolkit/Toolkit.WPF/UI/Controls/MeasureToolbar/MeasureToolbar.Theme.xaml

```diff
@@ -127,8 +127,6 @@
                                     Padding="8,4,8,4"
                                     ToolTip="Undo"
                                     Grid.Column="5"
-                                    DataContext="{TemplateBinding MapView}"
-                                    Command="{Binding Path=SketchEditor.UndoCommand}"
```

Comment:
Problem with this is you now lost the ability to enable/disable the button for when there is something to undo or not.
I'd instead suggest you create command properties on the toolbar you can bind here. Whether they can undo comes down to whether a measure is in process (meaning _geometryEditor is assigned if you make the change to only have it assigned while measure is in process) + geometry editor can undo (the first bit is important).